Only the pesky linter is not happy
```
./multimetric/cls/calc/halstead.py:298:20: C812 missing trailing comma
./multimetric/cls/calc/halstead.py:304:20: C812 missing trailing comma
./multimetric/cls/calc/maintenance.py:230:84: C812 missing trailing comma
./multimetric/cls/calc/maintenance.py:233:86: C812 missing trailing comma
./multimetric/cls/metric/cyclomatic.py:74:89: C812 missing trailing comma
./multimetric/cls/metric/cyclomatic.py:98:51: C812 missing trailing comma
```

--- a/multimetric/cls/calc/halstead.py
+++ b/multimetric/cls/calc/halstead.py
@@ -295,13 +295,13 @@ class MetricBaseCalcHalstead(MetricBaseCalc):
 
         """
         metrics[MetricBaseCalcHalstead.METRIC_HALSTEAD_VOLUME] = self._getVolume(
-            metrics
+            metrics,
         )
         metrics[
             MetricBaseCalcHalstead.METRIC_HALSTEAD_DIFFICULTY
         ] = self._getDifficulty(metrics)
         metrics[MetricBaseCalcHalstead.METRIC_HALSTEAD_EFFORT] = self._getEffort(
-            metrics
+            metrics,
         )
         metrics[MetricBaseCalcHalstead.METRIC_HALSTEAD_TIMEREQ] = self._getTime(metrics)
         metrics[MetricBaseCalcHalstead.METRIC_HALSTEAD_BUGS] = self._getBug(metrics)

--- a/multimetric/cls/calc/maintenance.py
+++ b/multimetric/cls/calc/maintenance.py
@@ -227,9 +227,9 @@ class MetricBaseCalcMaintenanceIndex(MetricBaseCalc):
         ] = MetricBaseCalcMaintenanceIndex.MI_METHOD[self.__miMethod](metrics)
         # Sanity
         metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX] = max(
-            metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX], 0
+            metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX], 0,
         )
         metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX] = min(
-            metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX], 100
+            metrics[MetricBaseCalcMaintenanceIndex.METRIC_MAINTAINABILITY_INDEX], 100,
         )
         return super().get_results(metrics)

--- a/multimetric/cls/metric/cyclomatic.py
+++ b/multimetric/cls/metric/cyclomatic.py
@@ -71,7 +71,7 @@ class MetricBaseCyclomaticComplexity(MetricBase):
         self._metrics[
             MetricBaseCyclomaticComplexity.METRIC_CYCLOMATIC_COMPLEXITY
         ] = max(
-            self._internalstore["conditions"] - self._internalstore["exitpoints"] + 2, 0
+            self._internalstore["conditions"] - self._internalstore["exitpoints"] + 2, 0,
         )
         return self._metrics
 
@@ -95,6 +95,6 @@ class MetricBaseCyclomaticComplexity(MetricBase):
             __exitPoints += x["exitpoints"]
         return {
             MetricBaseCyclomaticComplexity.METRIC_CYCLOMATIC_COMPLEXITY: max(
-                __conditions - __exitPoints + 2, 0
+                __conditions - __exitPoints + 2, 0,
             ),
         }

